lib/dashboard/home/tab_bar/tokens/send_receive_home/cubit/send_receive_home_cubit.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:altme/app/app.dart';
+import 'package:altme/dashboard/dashboard.dart';
+import 'package:altme/wallet/wallet.dart';
+import 'package:bloc/bloc.dart';
+import 'package:equatable/equatable.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'send_receive_home_cubit.g.dart';
+
+part 'send_receive_home_state.dart';
+
+class SendReceiveHomeCubit extends Cubit<SendReceiveHomeState> {
+  SendReceiveHomeCubit({
+    required this.client,
+    required this.walletCubit,
+  }) : super(const SendReceiveHomeState());
+
+  final DioClient client;
+  final WalletCubit walletCubit;
+
+  Future<void> init({String baseUrl = ''}) async {
+    try {
+      emit(state.loading());
+      final xtz = await getXtzBalance(
+        baseUrl,
+        walletCubit.state.currentAccount.walletAddress,
+      );
+      emit(state.copyWith(xtz: xtz));
+      final operations = await _getOperations(baseUrl);
+      emit(state.copyWith(operations: operations));
+
+      emit(state.success());
+    } catch (e, s) {
+      emit(state.error(messageHandler: MessageHandler()));
+      getLogger(runtimeType.toString()).e('error in init() e: $e, $s', e, s);
```

Comment:
It will be great if you can mention the cubit name while doing logging... it will be easier to track...For reference, you can take my other logging examples..

Reply:
@TalebRafiepour for all logging..

Reply:
final log = getLogger('ScanCubit - credentialOffer');
log.i('verifying Credential');
log.w('failed to verify credential', jsonVerification['errors']);
log.e('something went wrong - $e');

Reply:
@bibash28 the runtimeType.toString() returns the class name (Here is the Cubit name)